pygame_base_template.py

```diff
@@ -0,0 +1,115 @@
+"""
+ Pygame base template for opening a window
+
+ Sample Python/Pygame Programs
+ Simpson College Computer Science
+ http://programarcadegames.com/
+ http://simpson.edu/computer-science/
+
+ Explanation video: http://youtu.be/vRB_983kUMc
+"""
+
+
+
+import pygame
+from playermoves import *
+
+
+# Define some colors
+BLACK = (0, 0, 0)
+WHITE = (255, 255, 255)
+GREEN = (0, 255, 0)
+RED = (255, 0, 0)
+jumping = False
+pygame.init()
+
+# Set the width and height of the screen [width, height]
+size = (700, 500)
+screen = pygame.display.set_mode(size)
+
+pygame.display.set_caption("My Game")
+
+# Loop until the user clicks the close button.
+done = False
+
+# Used to manage how fast the screen updates
+clock = pygame.time.Clock()
+frames = 0;
+startjump = 0
+x_min = 0
+x_max = 200
+rectx = 100
+recty = 400
+
+# -------- Main Program Loop -----------
+while not done:
+
+    # --- Main event loop
+    
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+            done = True
+        #if event.type == pygame.KEYDOWN:

```

Comment:
Please remove comments when submitting the final code

Reply:
Check. “comments” == “commented-out code”, in this context.